spring-boot/admin/src/main/resources/templates/admin/fragments/addOrderingToList.html

```diff
@@ -0,0 +1,66 @@
+<html xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity4"
+      xmlns="http://www.w3.org/1999/xhtml">
+<body>
+
+<div th:fragment="addOrderingToPublicServicesList">
```

Comment:
The script looks totally generic, not related to public services. Maybe rename the fragment as `scriptToInitializeDataTableAndAddOrdering`

Reply:
Done. There is now a JS function named 'initializeDataTableAndAddOrdering' which is doing the job.